examples/direct_fidelity_estimation.ipynb

```diff
@@ -0,0 +1,228 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running the Direct Fidelity Estimation (DFE) algorithm\n",
+    "This example walks through the steps of running the direct fidelity estimation (DFE) algorithm as described in these two paper:\n",
+    "\n",
+    "* Direct Fidelity Estimation from Few Pauli Measurements (https://arxiv.org/abs/1104.4695)\n",
+    "* Practical characterization of quantum devices without tomography (https://arxiv.org/abs/1104.3835)\n",
+    "\n",
+    "The code is optimized for Clifford circuits using tableaux\n",
+    "* Improved Simulation of Stabilizer Circuits (https://arxiv.org/pdf/quant-ph/0406196.pdf)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0: ───@───Z───────\n",
+      "      │\n",
+      "1: ───┼───────X───\n",
+      "      │       │\n",
+      "2: ───X───H───@───\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Import Cirq, DFE, and create a circuit\n",
+    "\n",
+    "import cirq\n",
+    "import direct_fidelity_estimation as dfe\n",
```

Comment:
This import works when the notebook is run from the same directory as the example `direct_fidelity_estimation.py` in the examples directory; but won't work if this notebook is copied elsewhere.

In general, in Python I think these sorts of imports are discouraged in favor of using a proper package structure. Fixing this is well outside of the scope of the current PR. I have a feeling this project is eclipsing the bounds of a a simple "example". We can chat about various strategies to flesh this out better

Reply:
Added "examples." in front of the import